restful.module

```diff
@@ -561,3 +564,18 @@ function restful_date_time_format_element_validate($element, &$form_state) {
     )));
   }
 }
+
+/**
+ * Implements hook_restful_resource_alter().
+ */
+function restful_restful_resource_alter(ResourceInterface &$resource) {
```

Comment:
I believe that is a very tricky thing to do. You can play with module weights, but that may have other unintended consequences. I don't know how else this could be achieved.